frontend-public/src/components/dashboard/mine/variances/Variances.js

```diff
@@ -49,6 +68,80 @@ export class Variances extends Component {
     this.props.fetchVarianceStatusOptions();
   }
 
+  handlAddDocuments = (files, varianceGuid) => {
+    Promise.all(
```

Comment:
The promise isn't being returned, so this function will always return `undefined`, and the rest of the app will not be able to wait for this async operation to complete

Reply:
ahh, this explains so much.